<launch>

   <!-- Args -->
  <arg name="use_sim_time" default="true" doc="Use simulation time when replaying rosbags with '--clock' option."/>
  <arg name="rviz" default="true" doc="Visualize results with RViz."/>
  <arg name="gps" default="true" doc="If true, use GPS data to calibrate SLAM output. Otherwise, provide calibration."/>

  <!-- SLAM -->
  <include file="$(find lidar_slam)/launch/slam.launch">
    <arg name="use_sim_time" value="$(arg use_sim_time)"/>
    <arg name="rviz" value="$(arg rviz)"/>
    <arg name="gps" value="$(arg gps)"/>
  </include>

  <!-- Pose graph optimization using GPS and SLAM poses -->
  <node name="pose_graph_optimization" pkg="lidar_slam" type="pose_graph_optimization_node" output="screen">
    <rosparam file="$(find lidar_slam)/params/pose_graph_optim_config.yaml" command="load"/>
  </node>

</launch>